lib/sequent/core/base_event_handler.rb

```diff
@@ -47,5 +47,8 @@ def initialize(record_session = Sequent::Core::RecordSessions::ActiveRecordSessi
                      :find_records, :last_record
 
     end
+
+    # Alias the above class
+    Projector = BaseEventHandler
```

Comment:
Yes that's possible, but I didn't know the impact of that change. Make a decision and I'll update it ;)